src/Microsoft.Data.Sqlite.Core/SqliteConnection.cs

```diff
@@ -227,6 +227,33 @@ public override void Open()
             SetState(ConnectionState.Open);
         }
 
+        /// <summary>
+        /// Open a BLOB for incremental I/O.
+        /// </summary>
+        /// <param name="tableName">The table name.</param>
+        /// <param name="columnName">The column name.</param>
+        /// <param name="rowid">The row index.</param>
+        /// <param name="readWriteAccess">Flag indicating whether read and write access is desired (otherwise only read access).</param>
+        /// <returns>Object to access the BLOB.</returns>
+        public SqliteBlob OpenBlob(string tableName, string columnName, long rowid, bool readWriteAccess)
```

Comment:
I'd rather not add new API for this.

Instead, I'd like `SqliteDataReader.GetStream()`, `GetBytes()`, and `GetChars()` to start using `sqlite3_blob` if you happen to select `rowid` (or an alias)

Reply:
Okay, but this means that we would not provide the possibility to add an empty blob and write to it later (or do you want to have such a functionality included in GetStream?)!

Reply:
At the momnet I have no idea how to recognize, that id is an alias for rowid. Any suggestions?

Reply:
Hmm... Is there anything for that in the GetSchemaTable() code? I think you can assume that if there's a single `integer` primary key column on the table, that column aliases `rowid` (unless they created it `WITHOUT ROWID` in which case they can't use blol I/o)

Reply:
There may be something in here too: https://github.com/aspnet/EntityFrameworkCore/blob/dev/src/EFCore.Sqlite.Core/Scaffolding/Internal/SqliteDatabaseModelFactory.cs